code/modules/mob/living/carbon/human/species/_species.dm

```diff
@@ -761,105 +761,8 @@
 
 	return FALSE //Unsupported slot
 
-/datum/species/proc/get_perceived_trauma(mob/living/carbon/human/H)
```

Comment:
I really don't know why this all on the species datum when it's primarily a mob function.